ci/scripts/start_instance.sh

```diff
@@ -45,81 +45,85 @@ fi
+GCP_SUBNETWORK=${GCP_SUBNETWORK##*/}
+
+
 echo "${INSTANCE_NAME}" > "instance-data/instance-name"
-echo "${PROJECT}" > "instance-data/project"
+echo "${GCP_PROJECT}" > "instance-data/project"
 echo "${ZONE}" > "instance-data/zone"
 
 echo 'StrictHostKeyChecking no' >> /etc/ssh/ssh_config
 RAM_MEGABYTES=$( expr ${RAM} \* 1024 )
 
-while true; do
-    TTL=$(($(date +%s) + 60 * 60 * 6))
-    LABELS="instance_type=heavy-lifter,time-to-live=${TTL},job-name=${SANITIZED_BUILD_JOB_NAME},pipeline-name=${SANITIZED_BUILD_PIPELINE_NAME},build-name=${SANITIZED_BUILD_NAME}"
-
-    set +e
-    # Try to kill any existing machine before starting
-    gcloud compute --project=${PROJECT} instances delete ${INSTANCE_NAME} --zone=${ZONE} --delete-disks=all -q &>/dev/null
-
-    INSTANCE_INFORMATION=$(gcloud compute --project=${PROJECT} instances create ${INSTANCE_NAME} \
-      --zone=${ZONE} \
-      --machine-type=custom-${CPUS}-${RAM_MEGABYTES} \
-      --min-cpu-platform=Intel\ Skylake \
-      --network="heavy-lifters-2" \
-      --subnet="heavy-lifters-2" \
-      --image-family="${IMAGE_FAMILY_PREFIX}${WINDOWS_PREFIX}geode-builder" \
-      --image-project=${PROJECT} \
-      --boot-disk-size=100GB \
-      --boot-disk-type=pd-ssd \
-      --labels="${LABELS}" \
-      --format=json)
-    CREATE_EXIT_STATUS=$?
-    set -e
-
-    if [ ${CREATE_EXIT_STATUS} -eq 0 ]; then
-        break
-    fi
-
-    TIMEOUT=60
-    echo "Waiting ${TIMEOUT} seconds..."
-    sleep ${TIMEOUT}
-done
+TTL=$(($(date +%s) + 60 * 60 * 6))
+LABELS="instance_type=heavy-lifter,time-to-live=${TTL},job-name=${SANITIZED_BUILD_JOB_NAME},pipeline-name=${SANITIZED_BUILD_PIPELINE_NAME},build-name=${SANITIZED_BUILD_NAME}"
+
+INSTANCE_INFORMATION=$(gcloud compute --project=${GCP_PROJECT} instances create ${INSTANCE_NAME} \
+  --zone=${ZONE} \
+  --machine-type=custom-${CPUS}-${RAM_MEGABYTES} \
+  --min-cpu-platform=Intel\ Skylake \
+  --network="${GCP_NETWORK}" \
+  --subnet="${GCP_SUBNETWORK}" \
+  --image-family="${IMAGE_FAMILY_PREFIX}${WINDOWS_PREFIX}geode-builder" \
+  --boot-disk-size=100GB \
+  --boot-disk-type=pd-ssd \
+  --labels="${LABELS}" \
+  --tags="heavy-lifter" \
+  --format=json)
 
 echo "${INSTANCE_INFORMATION}" > instance-data/instance-information
 
-INSTANCE_IP_ADDRESS=$(echo ${INSTANCE_INFORMATION} | jq -r '.[].networkInterfaces[0].accessConfigs[0].natIP')
+INSTANCE_IP_ADDRESS=$(echo ${INSTANCE_INFORMATION} | jq -r '.[].networkInterfaces[0].networkIP')
 echo "${INSTANCE_IP_ADDRESS}" > "instance-data/instance-ip-address"
 
 if [[ -z "${WINDOWS_PREFIX}" ]]; then
-  while ! gcloud compute --project=${PROJECT} ssh geode@${INSTANCE_NAME} --zone=${ZONE} --ssh-key-file=${SSHKEY_FILE} --quiet -- true; do
+  SSH_TIME=$(($(date +%s) + 60))
+  echo -n "Attempting to SSH to instance."
+  while ! gcloud compute ssh geode@${INSTANCE_NAME} --zone=${ZONE} --internal-ip --ssh-key-file=${SSHKEY_FILE} --quiet -- true; do
+    if [ $(date +%s) > ${SSH_TIME} ]; then
```

Comment:
This syntax is not right - it should be `[[ ... > ...  ]]` or `[ ... -gt ... ]`. As it stands it's a redirection.